clients/wallet/src/stellar_wallet.rs

```diff
@@ -26,10 +26,7 @@ pub struct StellarWallet {
 }
 
 impl StellarWallet {
-	pub fn from_secret_encoded(
-		secret_key: &String,
-		is_public_network: bool,
-	) -> Result<Self, Error> {
+	pub fn from_secret_encoded(secret_key: &str, is_public_network: bool) -> Result<Self, Error> {
```

Comment:
Maybe we should create another function here called
```
from_secret_key(secret_key: SecretKey, is_public_network: bool) -> ....
```
that is very similar to the existing function but we don't have to go through the hassle of artificially creating strings all the time?